locales/hu_HU/LC_MESSAGES/duckduckgo.po

```diff
@@ -4164,6 +4152,7 @@ msgstr "Ennek az oldalnak szükségre van Javascriptre a működéshez."
 
 msgid "This video is not yet able to be watched here. You can watch it on %s."
 msgstr ""
+"Ezt a videót itt még nem lehet megnézni. Meg lehet nézni a (z)% s oldalon."
```

Comment:
@zoli111 Here the string must include the `%s` as is. Can you double check that `(z)% s`? Maybe a typo? Thanks!